<p>Don’t use the word ‘Class’ as part of class names or ‘Enum’ as part of enum class names.</p>

<h2>Noncompliant Code Example</h2>
<pre>
class myClass { }
class testclass{ }
enum sampleenum{ }
enum SampleEnum{ }
</pre>

<h2>Compliant Solution</h2>
<pre>
class student { }
enum Season{ }
</pre>
