<p>Don’t use one letter variable names (except ‘e’ for exceptions, ‘i, j’ in for loops)</p>

<h2>Noncompliant Code Example</h2>
<pre>
int i;
String l;
boolean p;
float k;
char a;
</pre>

<h2>Compliant Solution</h2>
<pre>
int number;
String name;
</pre>
